src/Command/BaseRokkaCliCommand.php

```diff
@@ -130,69 +90,40 @@ public function verifyStackExists($stackName, $organization, OutputInterface $ou
     }
 
     /**
-     * @param $organization
-     * @param OutputInterface $output
+     * Get a valid organization name.
      *
-     * @return bool
-     */
-    public function verifyOrganizationName($organization, OutputInterface $output)
-    {
-        if ($organization) {
-            return true;
-        }
-
-        $output->writeln($this->formatterHelper->formatBlock([
-            'Error!',
-            'The organization "'.$organization.'" is not valid!',
-        ], 'error', true));
-
-        return false;
-    }
-
-    /**
-     * @param $organization
-     * @param OutputInterface $output
-     * @param User|null       $client
+     * If an organization name is provided, make sure it is valid and actually exists in the API.
+     * Otherwise return the default organization name.
      *
-     * @return bool
+     * @param string          $organizationName The organization name or an empty value if none is specified
+     * @param OutputInterface $output           Console to write information for the user
+     *
+     * @return string|bool The organization name to use or false if the provided name is not valid.
      */
-    public function verifyOrganizationExists($organization, OutputInterface $output, User $client = null)
+    public function resolveOrganizationName($organizationName, OutputInterface $output)
     {
-        if (!$organization) {
-            return false;
+        if (!$organizationName) {
+            return $this->rokkaHelper->getDefaultOrganizationName();
```

Comment:
Given the return description, here you're returning the default organization, that could not exist